Maddori.Apple/Maddori.Apple/Screen/MyFeedback/MyFeedbackEdit/MyFeedbackEditViewController.swift

```diff
@@ -90,9 +102,29 @@ final class MyFeedbackEditViewController: AddFeedbackViewController {
         }
     }
     
+    private func notChangedStartContent() -> Bool {
+        if let start = self.feedbackDetail.start {
+            if super.feedbackStartTextView.text == TextLiteral.addFeedbackViewControllerStartTextViewPlaceholder {
+                return true
+            } else {
+                if super.feedbackStartTextView.text == start {
+                    return true
+                } else {
+                    return false
+                }
+            }
```

Comment:
p3;
```suggestion
            if super.feedbackStartTextView.text == TextLiteral.addFeedbackViewControllerStartTextViewPlaceholder || start {
                return true
            } else {
                return false
            }
```

혹시 요롷게 안한 이유가 있으신가요??!

만약 일부러 `TextLiteral.addFeedbackViewControllerStartTextViewPlaceholder`와 `start` 를 분리시키신거라면,

```suggestion
            if super.feedbackStartTextView.text == TextLiteral.addFeedbackViewControllerStartTextViewPlaceholder {
                return true
            } else {
                return super.feedbackStartTextView.text == start ? true : false
            }
```
요롷게도 될 것 같아요!!

Reply:
그러네요 그러네요 !
한 집중 사이클에 빡 한다고 놓친것 같네요 ! 감사합니다 ^________^